app/html/sub/settings.html

```diff
@@ -44,9 +44,30 @@ <h1>@{_('Editing settings of <a href="%(url)s">%(sub)s</a>', url=url_for('sub.vi
         </div>
 
         <div class="pure-control-group">
-          <label for="polling" class="pure-checkbox">
-            @{editsubform.polling(checked=True if metadata.get('allow_polls') else False)!!html}
-            @{editsubform.polling.label.text}
+          <label for="text_posts" class="pure-checkbox">
+            @{editsubform.allow_text_posts(checked=True if metadata.get('allow_text_posts') else False)!!html}
+            @{editsubform.allow_text_posts.label.text}
+          </label>
+        </div>
+
+        <div class="pure-control-group">
+          <label for="link_posts" class="pure-checkbox">
+            @{editsubform.allow_link_posts(checked=True if metadata.get('allow_link_posts') else False)!!html}
+            @{editsubform.allow_link_posts.label.text}
+          </label>
+        </div>
+
+        <div class="pure-control-group">
+          <label for="upload_posts" class="pure-checkbox">
```

Comment:
The `for` in these label fields is wrong, it should match the name of the field (otherwise clicking the label won't tick the checkbox).